extensions/amp-story/1.0/amp-story-page-attachment.css

```diff
@@ -83,3 +83,113 @@
+}
+
+.i-amphtml-amp-story-page-attachment-ui-v2.i-amphtml-story-page-attachment-remote {
+  height: 56px !important;
+  --i-amphtml-outlink-cta-background-color: white !important;
+  --i-amphtml-outlink-cta-text-color: black !important;
+}
+
+[theme="dark"].i-amphtml-amp-story-page-attachment-ui-v2.i-amphtml-story-page-attachment-remote {
+  --i-amphtml-outlink-cta-background-color: black !important;
+  --i-amphtml-outlink-cta-text-color: white !important;
+}
+
+.i-amphtml-amp-story-page-attachment-ui-v2.amp-story-draggable-drawer-root.i-amphtml-story-draggable-drawer-open {
+  /* Faster duration for outlink drawer. */
+  transition-duration: .3s !important;
+}
+
+.i-amphtml-amp-story-page-attachment-ui-v2 .i-amphtml-story-page-attachment-remote-content {
+  height: 56px !important;
+  padding-inline-start: 12px !important;
+  padding-inline-end: 18px !important;
+  background: var(--i-amphtml-outlink-cta-background-color) !important;
+}
+
+.i-amphtml-amp-story-page-attachment-ui-v2 .i-amphtml-story-page-attachment-remote-img {
+  width: 32px !important;
+  height: 32px !important;
+  flex-shrink: 0 !important;
+  padding: 0px !important;
+  border-radius: 50% !important;
+  background-size: cover !important;
+  background-repeat: no-repeat !important;
+  overflow: hidden !important;
+}
+
+.i-amphtml-amp-story-page-attachment-ui-v2 svg.i-amphtml-story-page-open-attachment-link-icon {
+  fill: var(--i-amphtml-outlink-cta-text-color) !important;
+}
+
+[cta-image="none"].i-amphtml-amp-story-page-attachment-ui-v2 .i-amphtml-story-page-attachment-remote-img {
+  display: none !important;
+}
+
+.i-amphtml-amp-story-page-attachment-ui-v2 .i-amphtml-story-page-attachment-remote-title {
+  font-size: 14px !important;
+  padding: 0 12px !important;
+  color: var(--i-amphtml-outlink-cta-text-color) !important;
+  /* For RTL support. */
+  display: flex !important;
+}
+
+.i-amphtml-amp-story-page-attachment-ui-v2 .i-amphtml-story-page-attachment-remote-title :first-child {
+  font-weight: bold !important;
+}
+
+.i-amphtml-amp-story-page-attachment-ui-v2 .i-amphtml-story-page-attachment-remote-title :first-child:after {
+  /* For RTL supportted space. */
+  content: "\00a0 " !important;
+}
+
+.i-amphtml-amp-story-page-attachment-ui-v2 .i-amphtml-story-page-attachment-remote-title :nth-child(2) {
+  white-space: nowrap !important;
+  overflow: hidden !important;
+  text-overflow: ellipsis !important;
+}
+
+.i-amphtml-amp-story-page-attachment-ui-v2 .i-amphtml-story-page-attachment-remote-icon {
+  margin-inline-start: auto !important;
+  width: 20px !important;
+  height: 20px !important;
+  flex-shrink: 0 !important;
+  fill: var(--i-amphtml-outlink-cta-text-color) !important;
+  background: none !important;
+}
+
+.i-amphtml-amp-story-page-attachment-ui-v2.i-amphtml-story-page-attachment-remote.i-amphtml-story-draggable-drawer-open:before {
+  content: "" !important;
+  position: absolute !important;
+  width: 100% !important;
+  height: 2px !important;
```

Comment:
Note: Is hard to see currently, specially when the color matches the chrome UI (left a screenshot below of my phone)

<img src="https://user-images.githubusercontent.com/22420856/116105395-472b2780-a67f-11eb-9dc9-c04136c7c208.png" width="200">

Reply:
Double checked the mocks and it's actually 4px which helps visibility a lot. Demo is updated.